<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>918</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">kOferta</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/zasoby.qrc">
    <normaloff>:/ico</normaloff>:/ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tabela zamówienia</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="addTowar">
            <property name="text">
             <string>Dodaj Towar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rabat">
            <property name="text">
             <string>Ustaw rabat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="delw">
            <property name="text">
             <string>Usuń wiersz</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="MerchandiseListView" name="tableView" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Pozostałe informacje</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QCommandLinkButton" name="commandLinkButton_klient">
            <property name="text">
             <string>Wybór Klienta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_klient">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="lineWrapMode">
             <enum>QPlainTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="2" rowspan="2">
         <widget class="QGroupBox" name="waluta">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Waluta:</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0" colspan="3">
            <widget class="QRadioButton" name="radioButton_EUR">
             <property name="text">
              <string>Euro</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QRadioButton" name="radioButton_PLN">
             <property name="text">
              <string>Złotówki</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_exchangeRate">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Kurs:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="2">
            <widget class="QDoubleSpinBox" name="spinBox_exchangeRate">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="decimals">
              <number>4</number>
             </property>
             <property name="maximum">
              <double>99.999899999999997</double>
             </property>
             <property name="singleStep">
              <double>0.010000000000000</double>
             </property>
             <property name="value">
              <double>4.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QGroupBox" name="grupBox_zapytanie">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Zapytanie</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="sizeConstraint">
              <enum>QLayout::SetDefaultConstraint</enum>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QCheckBox" name="checkBox_zapytanieData">
                 <property name="text">
                  <string>Data zapytania:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_zapytanieData">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_zapytanieData">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset resource="res/zasoby.qrc">
                   <normaloff>:/calendar</normaloff>:/calendar</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <widget class="QCheckBox" name="checkBox_zapytanieNr">
                 <property name="text">
                  <string>Numer zapytania:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_zapytanieNr">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>200</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>500</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="baseSize">
                  <size>
                   <width>500</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="readOnly">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_zapytanie">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" rowspan="2" colspan="2">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Pozostałe informacje</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0" colspan="2">
            <widget class="QCommandLinkButton" name="commandLinkButton_shipingTerms">
             <property name="text">
              <string>Warunki Dostawy</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPlainTextEdit" name="plainTextEdit_dostawa">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCommandLinkButton" name="commandLinkButton_shipmentTime">
             <property name="text">
              <string>Termin Dostawy</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPlainTextEdit" name="plainTextEdit_termin">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QCommandLinkButton" name="commandLinkButton_paymentTerms">
             <property name="text">
              <string>Warunki Płatności</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPlainTextEdit" name="plainTextEdit_platnosc">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QCommandLinkButton" name="commandLinkButton_offerTerms">
             <property name="text">
              <string>Warunki Oferty</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QPlainTextEdit" name="plainTextEdit_oferta">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="undoRedoEnabled">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QCommandLinkButton" name="commandLinkButton_other">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Uwagi</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QPlainTextEdit" name="plainTextEdit_uwagi">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QGroupBox" name="kolumny">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Układ wydruku:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QCheckBox" name="kol_lp">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>LP</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kol_towar">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Towar</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kol_specyfikacja">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Specyfikacja</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kol_cenaKat">
             <property name="text">
              <string>Cena katalogowa</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kol_cenaPln">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Cena kat. PLN</string>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kol_rabat">
             <property name="text">
              <string>Rabat</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kol_cena">
             <property name="text">
              <string>Cena</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kol_ilosc">
             <property name="text">
              <string>Ilość</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="kol_wartosc">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Wartość</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="2" rowspan="2">
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>430</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
       <zorder>grupBox_zapytanie</zorder>
       <zorder>kolumny</zorder>
       <zorder>waluta</zorder>
       <zorder>verticalSpacer_7</zorder>
       <zorder>groupBox</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1134</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOferta">
    <property name="title">
     <string>Oferta</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionNR"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionDo_PDF"/>
    <addaction name="actionDruk"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionO_kOferta"/>
    <addaction name="actionO_Qt"/>
   </widget>
   <widget class="QMenu" name="menuKlient">
    <property name="title">
     <string>Baza Danych</string>
    </property>
    <addaction name="klientNowy"/>
    <addaction name="klientEdycja"/>
    <addaction name="separator"/>
    <addaction name="actionNowy_Towar"/>
    <addaction name="separator"/>
    <addaction name="actionDodaj_warunki"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>kOferta</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="action_changePassword"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuConnect"/>
   <addaction name="menuOferta"/>
   <addaction name="menuExport"/>
   <addaction name="menuKlient"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>Nowa</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Wyjście</string>
   </property>
  </action>
  <action name="actionDo_PDF">
   <property name="text">
    <string>PDF</string>
   </property>
  </action>
  <action name="actionDo_HTML">
   <property name="text">
    <string>HTML</string>
   </property>
  </action>
  <action name="actionDruk">
   <property name="text">
    <string>Druk</string>
   </property>
  </action>
  <action name="actionO_kOferta">
   <property name="text">
    <string>O kOferta</string>
   </property>
  </action>
  <action name="actionO_Qt">
   <property name="text">
    <string>O Qt</string>
   </property>
  </action>
  <action name="actionNR">
   <property name="text">
    <string>Nadaj nowy numer</string>
   </property>
  </action>
  <action name="klientNowy">
   <property name="text">
    <string>Nowy Klient</string>
   </property>
  </action>
  <action name="klientImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="klientEksport">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eksport</string>
   </property>
  </action>
  <action name="klientEdycja">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edycja Klienta</string>
   </property>
  </action>
  <action name="towarNowy">
   <property name="text">
    <string>Nowy</string>
   </property>
  </action>
  <action name="towarImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="towarEksport">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eksport</string>
   </property>
  </action>
  <action name="towarEdycja">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Edycja</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Połącz</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Rozłącz</string>
   </property>
  </action>
  <action name="actionNowy_Towar">
   <property name="text">
    <string>Nowy Towar</string>
   </property>
  </action>
  <action name="actionDodaj_warunki_dostawy">
   <property name="text">
    <string>Dodaj warunki dostawy</string>
   </property>
  </action>
  <action name="actionDodaj_termin_dostawy">
   <property name="text">
    <string>Dodaj termin dostawy</string>
   </property>
  </action>
  <action name="actionDodaj_warunki_platnosci">
   <property name="text">
    <string>Dodaj warunki platnosci</string>
   </property>
  </action>
  <action name="actionDodaj_warunki">
   <property name="text">
    <string>Dodaj warunki</string>
   </property>
  </action>
  <action name="actionUstawienia">
   <property name="text">
    <string>Ustawienia</string>
   </property>
  </action>
  <action name="action_changePassword">
   <property name="text">
    <string>Zmień hasło</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MerchandiseListView</class>
   <extends>QWidget</extends>
   <header>MerchandiseListView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res/zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
